cartridges/custom_cartridge_1/cartridge/templates/default/templateTaskSession.isml

```diff
@@ -0,0 +1,8 @@
+<isif condition="${session.custom.newVar !== 1}">
```

Comment:
Looks like his if/else clause only increment if newVar is equal to 1 and then reset it to one again?

Reply:
fixed